src/components/MenuItem/styles.module.css

```diff
@@ -4,6 +4,7 @@
 @value borderDarkGray from layout;
 
 .highlightLabel {
+<<<<<<< HEAD
```

Comment:
This looks like it's left over from a resolved merge-conflict, should be removed.